torch_xla/csrc/ops/flash_attention_forward.cpp

```diff
@@ -235,13 +235,25 @@ void custom_call_flash_attention_forward(cudaStream_t stream, void** buffers,
   // inference.
   const int num_m_blocks = (params.seqlen_q + 64 - 1) / 64;
   launch_params.num_splits = 0;
-  if (1 - params.p_dropout == 0.0f) {  // SplitKV is not implemented for dropout
+  if (params.p_dropout == 1.0f) {  // SplitKV is not implemented for dropout
     if (launch_params.num_splits < 1) {
```

Comment:
Should be params.num_splits here? `launch_params.num_splits = 0` in L237.

Reply:
Done